ui/components/pr/pr.go

```diff
@@ -127,9 +127,15 @@ func (pr *PullRequest) renderLines() string {
 		deletions = pr.Data.Deletions
 	}
 
-	return pr.getTextStyle().Render(
-		fmt.Sprintf("%d / -%d", pr.Data.Additions, deletions),
-	)
+    return pr.getTextStyle().Render(
+        components.KeepSameSpacesOnAddDeletions(
+            fmt.Sprintf(
+                "\033[32m+%s \033[31m-%s",
```

Comment:
You should use `lipgloss` for this. Specifically `lipgloss.NewStyle().Foreground(lipgloss.Color("1"))` and then use `lipgloss.JoinHorizontal()`

Reply:
Thanks for the review @dlvhdr 
i updated the method like this : 
```golang
func (pr *PullRequest) renderLines() string {
	deletions := 0
	if pr.Data.Deletions > 0 {
		deletions = pr.Data.Deletions
	}

    greenStyle := lipgloss.NewStyle().Foreground(lipgloss.Color("2"))
    redStyle := lipgloss.NewStyle().Foreground(lipgloss.Color("1"))

    greenText := greenStyle.Render("+" + components.FormatNumber(pr.Data.Additions))
    redText := redStyle.Render("-" + components.FormatNumber(deletions))

    return pr.getTextStyle().Render(
        components.KeepSameSpacesOnAddDeletions(
            lipgloss.JoinHorizontal(lipgloss.Top, greenText, redText),
        ),
    )
}
```

but am having an output not "fine" for the redText : 

![Peek 2023-07-23 15-55](https://github.com/dlvhdr/gh-dash/assets/22576758/64b99ccf-498a-41ea-accb-af839760225f)


Any tips ? 🤔

Reply:
Not near a computer right now but look at the code, there are other instances where I applied the "selected" background again. When you create a style you lose the bg the cell previously had, so you need to reapply it.